packages/react-native/src/Authenticator/__tests__/withAuthenticator.spec.tsx

```diff
@@ -53,7 +53,9 @@ describe('withAuthenticator', () => {
 
     const { toJSON, queryByTestId } = render(<TestApp />);
 
-    expect(queryByTestId(CHILD_TEST_ID)).toBeNull();
+    waitFor(() => {
```

Comment:
Same here, intentional?

Reply:
Yep, cleanup of `Warning 'An update inside a test was not wrapped in act(...)'` console errors with `@testing-library/react`